chalicelib_fourfront/checks/helpers/wfr_utils.py

```diff
@@ -206,7 +206,8 @@
     'sci-Hi-C': [''],
     'MC-3C': [''],
     'MC-Hi-C': [''],
-    'Hi-ChIP': [''],
+    # Preliminary - Released to network  # NO-NORM
```

Comment:
I'm not sure about the "Released to network". This needs to be reviewed, and maybe you might need to edit the comments also on the other targeted-Hi-C types.

Additionally, you might move this block right below plac-seq so that they are all together.

Reply:
Agreed on moving the location so all the similar types are grouped together.  I think the 'released to network' comments are not quite accurate.  You might check with @clarabakker to see if these should be removed or updated.  They probably correspond to the various parameters in the checks and/or actions and reflect things like whether the generated files are automatically moved to the processed file tab or if they are kept in supplementary files and others like normalization or if there is an enzyme file used.  In any case Hi-ChIP should be treated exactly as Plac-seq so as long as you used that check/action as the template and didn't change any parameters might as well keep the same comment as for that pipeline and then do a wholesale review of the comments in a separate PR after getting Clara's input.

Reply:
Ok, moving the Hi-ChIP block below plac-seq, keeping the comment as it is for now. I will check with Clara to update in another PR.

Reply:
Yes, the specifications in the comment should be determined by the parameters used in the hichip_status and hichip_start functions. Parameters for omitting restriction enzymes and normalization are in the call to [wfr_utils.check_hic()](https://github.com/4dn-dcic/foursight/blob/a9737bfc838790fd86af38c0985bf12b074f31d5/chalicelib_fourfront/checks/wfr_checks.py#L1290), so NO-NORM is correct here.

"Preliminary - Released to network" should be used depending on the `move_to_pc` parameter of [wfr_utils.start_tasks](https://github.com/4dn-dcic/foursight/blob/a9737bfc838790fd86af38c0985bf12b074f31d5/chalicelib_fourfront/checks/wfr_checks.py#L1309). Since `move_to_pc` is true, the processed files will not be preliminary, so I think this "Preliminary - Released to network" can be removed (unless we have any reason we would want to restrict the files). PLAC-seq is the same (not preliminary).